catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -296,6 +375,51 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
                 .collect(toSet());
     }
 
+    @SuppressWarnings("unchecked")
+    private List<MetacardValidator> getMetacardValidators(
+            MetacardValidatorDefinition validatorDefinition) {
+        List<MetacardValidator> metacardValidators = new ArrayList<>();
+
+        MetacardValidator metacardValidator = null;
+        String validatorName = null;
+        Object metacardValidatorObj = validatorDefinition.arguments.get(VALIDATOR_PROPERTY);
+        if (metacardValidatorObj instanceof String) {
+            validatorName = (String) metacardValidatorObj;
+        }
+        if (validatorName != null) {
+            String metacardType = validatorDefinition.metacardType;
+            switch (validatorName) {
+            case REQUIRED_ATTRIBUTE_VALIDATOR_PROPERTY: {
```

Comment:
I stubbed this in like this with the anticipation that more MetacardValidators would be added in the future.